src/progressbar/styled-components.js

```diff
@@ -0,0 +1,42 @@
+/*
+Copyright (c) 2018 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+import {styled} from '../styles/index';
+export const Root = styled('div', props => {
+  return {
+    width: '100%',
+  };
+});
+
+export const Bar = styled('div', props => {
+  const {$theme, $value, $successValue} = props;
+  const {colors, sizing, borders} = $theme;
+  let background = 'linear-gradient(to right';
+  const fillColor = colors.primary400;
+  const emptyColor = colors.mono400;
+  background += ', ';
+  background += fillColor;
+  background += ' ' + ($value / $successValue) * 100 + '%, ';
+  background += emptyColor;
+  background += ' 0% ';
+  background += ', ' + emptyColor + ' 0%';
+  background += ')';
+  return {
+    margin: sizing.scale500,
+    borderRadius: borders.useRoundedCorners ? sizing.scale0 : '0',
+    background: background,
+    height: '4px',
+  };
+});
+
+export const Label = styled('div', props => {
+  return {
+    textAlign: 'center',
+    ...props.$theme.typography.font250,
+    color: props.$theme.colors.mono700,
+  };
+});
```

Comment:
Can we center the slider value directly over the thumb?
![image](https://user-images.githubusercontent.com/875591/46685357-304c8480-cbaa-11e8-895f-55567f811576.png)

Reply:
haha yeaaaah, reviewing two PRs at once is dangerous, sorry about that. resolving here.